### ISSUE TITLE
leftover apache.pid file prevents start of container

### ISSUE THREAD

**Issue by ltalirz** [materialscloud-org/tools-barebone#26]
We had an incident where likely some sort of hard reboot occurred.
Since the apache PID file was still left at `/var/run/apache2/apache2.pid`, the container did not properly start:
```
2021-05-25T13:53:15.678101635Z app[web.1]: httpd (pid 30) already running
2021-05-25T13:53:16.701874422Z app[web.1]: AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.3. Set the 'ServerName' directive globally to suppress this message
2021-05-25T13:53:16.712602196Z app[web.1]: httpd (pid 30) already running
```

Workaround:
```
dokku enter seekpath
$ rm /var/run/apache2/apache2.pid
$ exit
docker restart seekpath.web.1
```

This should probably be cleaned up at container start

mentioning @dou-du @giovannipizzi

**Comment by giovannipizzi** [comment]
@asle85 and @dou-du could you see if there is an easy way to fix this on this repository?

**Comment by asle85** [comment]
I think that just uncommenting [this line](https://github.com/materialscloud-org/tools-barebone/blob/d14e6b818f06625a4208269e56c5b2fd60a1635a/.docker_files/apache_run.sh#L19) should solve it, as it will run before the  apache service starts at container restart?

**Comment by ltalirz** [comment]
looks like it, please do!

**Comment by giovannipizzi** [comment]
Apparently I already knew the solution three years ago, I just decided to let you discover it :-P

**Comment by asle85** [comment]
solved in PR #27
